### DISCUSSION TITLE
[V3] Using navigate and redirects to URLs for other origin causes error

### DISCUSSION THREAD

**Discussion by epessine** [livewire/livewire#6448] [7. Bugs (archived)]
### Livewire version

3.0.0-beta.10

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

The problem occurs when you navigate URLs using `wire:navigate`, and then tries to use `$this->redirect()` from a Livewire component. If you try to redirect the user to a URL from a different origin (without using navigate), the console throws an error and breaks the application/navigation history:

```
Uncaught (in promise) DOMException: Failed to execute 'pushState' on 'History': A history state object with URL 'https://wolf-effertz.app.test/dashboard' cannot be created in a document with origin 'https://dev.app.test' and URL 'https://dev.app.test/user/profile'.
```

1. Navigate between pages using links with `wire:navigate`.
2. Try to call `redirect()` method on a Livewire component to a different origin URL and `navigate` set to `false`.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by onemoreahmad** [comment]
+1 
Had same issue since v3 came out, [mentioned it](https://github.com/livewire/livewire/discussions/5861) already [multiple times](https://github.com/livewire/livewire/discussions/5803#discussioncomment-6774395) with no luck.

**Comment by alejandrozepeda** [comment]
Same issue here, when redirecting to an external route with `$this->redirect()` or `redirect()->away()`

**Comment by luanfreitasdev** [comment]
+1

**Comment by joshhanley** [comment]
Thanks for reporting! Could someone please submit a failing test PR so we can investigate? Thanks!

**Comment by sammyaxe** [comment]
any update on this one?

**Reply by onemoreahmad** [reply]
make sure you set the full url with http in the redirect.

`$this->redirect(http....)`

**Reply by sammyaxe** [reply]
what about when I simply have `<a href="https://external.com" wire:navigate>`

**Reply by onemoreahmad** [reply]
You can't do that as far as I know. can't you juse delete this `wire:navigate` part when you add an external links ? 
Because it's useless to use wire:navigate and external link. you can't use Ajax between two different origins/domains. wire:navigate and ajax only (as a fact) works on the same domain. Any one please correct me if that's not true.

**Reply by sammyaxe** [reply]
yeah, but probably would be good to have logic in wire:navigate to simply ignore external links as now I have to build in additional logic for something pretty basic

**Reply by onemoreahmad** [reply]
Aha, I see what you're facing and I agree
